Source/Core/DolphinQt/TAS/GCTASInputWindow.cpp

```diff
@@ -84,6 +77,40 @@ GCTASInputWindow::GCTASInputWindow(QWidget* parent, int num) : TASInputWindow(pa
   setLayout(layout);
 }
 
+void GCTASInputWindow::SetTriggersBox()
+{
+  m_triggers_box = new QGroupBox(tr("Triggers"));
+
+  const QKeySequence l_shortcut_key_sequence = QKeySequence(Qt::ALT + Qt::Key_N);
+  const QKeySequence r_shortcut_key_sequence = QKeySequence(Qt::ALT + Qt::Key_M);
+
+  auto* l_label =
+      new QLabel(tr("Left (%1)").arg(l_shortcut_key_sequence.toString(QKeySequence::NativeText)));
+  auto* r_label =
+      new QLabel(tr("Right (%1)").arg(r_shortcut_key_sequence.toString(QKeySequence::NativeText)));
+
+  QBoxLayout* l_layout = new QHBoxLayout;
+  l_layout->addWidget(l_label);
+  QBoxLayout* r_layout = new QHBoxLayout;
+  r_layout->addWidget(r_label);
+
+  SliderValuePair* l_sliderValuePair = CreateSliderValuePair(
+      l_layout, 0, 255, l_shortcut_key_sequence, Qt::Horizontal, m_triggers_box);
+  SliderValuePair* r_sliderValuePair = CreateSliderValuePair(
+      r_layout, 0, 255, r_shortcut_key_sequence, Qt::Horizontal, m_triggers_box);
+
+  // Create grid layout
+  QGridLayout* triggers_layout = new QGridLayout;
+  triggers_layout->addWidget(l_label, 0, 0);
+  triggers_layout->addWidget(l_sliderValuePair->slider, 0, 1);
+  triggers_layout->addWidget(l_sliderValuePair->value, 0, 2);
+  triggers_layout->addWidget(r_label, 1, 0);
+  triggers_layout->addWidget(r_sliderValuePair->slider, 1, 1);
+  triggers_layout->addWidget(r_sliderValuePair->value, 1, 2);
+
+  m_triggers_box->setLayout(triggers_layout);
```

Comment:
```suggestion
  QBoxLayout* const l_slider_value_pair_layout = new QHBoxLayout;
  QBoxLayout* const r_slider_value_pair_layout = new QHBoxLayout;

  m_l_trigger_value = CreateSliderValuePair(
      l_slider_value_pair_layout , 0, 255, l_shortcut_key_sequence, Qt::Horizontal, m_triggers_box);
  m_r_trigger_value = CreateSliderValuePair(
      r_slider_value_pair_layout , 0, 255, r_shortcut_key_sequence, Qt::Horizontal, m_triggers_box);

  QFormLayout* const triggers_layout = new QFormLayout;
  triggers_layout->addRow(l_label, l_slider_value_pair_layout);
  triggers_layout->addRow(r_label, r_slider_value_pair_layout);

  m_triggers_box->setLayout(triggers_layout);
```

I would recommend using QFormLayout instead of QGridLayout.

- It more clearly represents the intended meaning of the layout. A form has two columns (labels and values) with a consistent dividing line between the two, whereas a grid represents any arbitrary gridlike pattern.
- Since you insert x_slider_value_pair_layout into the form instead of the slider and spinbox individually you no longer need the SliderValuePair struct and can revert all the changes made to accomodate that, simplifying the PR.